docs/admin/admin-ui/introduction.md

```diff
@@ -0,0 +1,34 @@
+---
+tags:
+- installation
+- license
+---
+
+# Overview
+
+Gluu Flex Admin UI is a reactive web interface to simplify the management and configuration of your Auth Server. It is one of the key services of Gluu Flex that enables viewing and editing configuration properties, interception scripts, clients, users, metrics, etc at one place.
```

Comment:
`reactive` is a technical term may be. If it is not going to be useful to the reader, you can remove it.

Reply:
removed